### ISSUE TITLE
fontforge moduel in python 2.7 wouldn't load some font files that actually work and get installed on windows

### ISSUE THREAD

**Issue by xandervx** [fontforge/fontforge#4212]
Fontforge from python wouldn't load some font files and throw the error.
`EnvironmentError: Open failed`
```
fontforge.open('font.ttf')
```
  '20170924' fontforge version 
Ubuntu 18.04 (wsl)

You can check the files here that wouldn't open here.

[nonloaded.zip](https://github.com/fontforge/fontforge/files/4300037/nonloaded.zip)

**Comment by skef** [comment]
Assuming you have legal permission to open these rights-protected fonts, you can pass the 1 flag to open, as in

```
fontforge.open('font.ttf', 1)
```

to indicate that permission to FontForge.

**Comment by xandervx** [comment]
oh thank you! so that was the issue? I saw that there are "some" flags but is there a description where they are described?

**Comment by skef** [comment]
Unfortunately the flags are currently only described in the native scripting language document. I've added the docs label to this issue to remind us to copy that info to the python docs.

When FontForge opens a font it often converts the tables into OpenType equivalents as best it can. These are somewhat accessible through the methods with `lookup` in their names. But you may find it easier to use `generateFeatureFile()` to output them and mergeFeature to reload them.

**Comment by xandervx** [comment]
I managed to find this. I suspect that is what you mean. https://fontforge.org/en-US/documentation/scripting/native/scripting-alpha/#Open
But I cannot locate the flags for opening a file. Only one example features `open(font, 1)` here. I want to also suppress the output to the command line (which kills my python kernel), hence my questions.

**Comment by skef** [comment]
The names of all the problematic fonts in your two archives start with either a numeral, unusual character, or "A". So it appears that you're trying to open every font in some huge archive with FontForge. The one that's failing that you've quoted is called "American_Brewery_Rough_Demo.ttf". The project folks will look at crashes and such (at some future point), but it's probably going to be easier to just ignore files that don't work at all.

The command line argument `-quiet` will remove some error messages but not all of them.

**Comment by xandervx** [comment]
Thank you. That's what I did. Yes, I was doing bulk validation of hundreds of fonts and fontforge is doing a great job and swiftly munching through them all. But them all of a sudden it chokes on something without a distinct reason why exactly at that particular font. I also have a 32gb machine and close every font after reading and validating it so memory and the flow isn't an issue. Sometimes just by removing one of the fonts it works. But then it might (or might not) choke at another one with around the same location (if you restart from the beginning). I worked around it by simply moving those fonts elsewhere and eventually it did work. It's a bit hacky but I was really lost with that crushing.
